OneCommunityGlobal/HighestGoodNetworkApp
# Vansh add floating and dismissable team notification modal

# Description
<img width="564" alt="Screen Shot 2023-11-15 at 5 39 31 PM" src="https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/47534216/803868d1-c720-4771-b981-22f5b075e3b3">

This PR is regarding a Summary Submission reminder to users on Thursday. This is the partial implementation (Part 1) of the feature. 
In Part 1, if a user logs into the application on Thursday, the Summary bar should display this message "You still need to complete the weekly summary, by FRIDAY at midnight (Pacific Time), for your manager's review Click here to submit it." instead of the old summary reminder message as seen below.
<img width="727" alt="Screen Shot 2023-11-15 at 5 24 38 PM" src="https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/47534216/a0684fd5-a9fb-423c-a295-bbbbed5d79bb">
Things to do: 
1. Implement a check for the User's role. This message should only be visible to Volunteers and assistant managers who are in a team.
2. Part 2: Display a floating modal of the message that gets displayed the first time a user logs in on Thursday, which is dismissable and then reduces to a simple message.


Fixes # (bug list priority: Urgent Dashboard Component.1)
Or Implements # (WBS) 

## Main changes explained:
- Implemented a method to check the current date of the user. If the day was Thursday (=5) then the text displayed was different than default.

## How to test:
1. check into the current branch
3. do `npm install` and `...` to run this PR locally
4. Set your current date to any Thursday (or Test this PR on Thursday)
5. Log in as a Volunteer/Assistant Manager on a team 
6. Go to dashboard
7. Verify that the summary reminder message displayed on the screen is "You still need to complete the weekly summary, by FRIDAY at midnight (Pacific Time), for your manager's review Click here to submit it."

## Screenshots or videos of changes:


## Note:
This PR is not for review and has been created to hand over the work to another developer.
